sources/api/schnauzer/src/helpers.rs

```diff
@@ -1301,6 +1366,178 @@ pub fn etc_hosts_entries(
     Ok(())
 }
 
+/// This helper writes out the default OCI runtime spec.
+///
+/// Default settings are specified in the following files:
+/// * sources/models/shared-defaults/oci-resource-limits.toml
+/// * sources/models/shared-defaults/oci-capabilities.toml
+
+pub fn oci_defaults(
+    helper: &Helper<'_, '_>,
+    _: &Handlebars,
+    _: &Context,
+    renderctx: &mut RenderContext<'_, '_>,
+    out: &mut dyn Output,
+) -> Result<(), RenderError> {
+    // To give context to our errors, get the template name (e.g. what file we are rendering), if available.
+    trace!("Starting oci_defaults helper");
+    let template_name = template_name(renderctx);
+    trace!("Template name: {}", &template_name);
+
+    // Check number of parameters, must be exactly two (OCI spec section to render and settings values for the section)
+    trace!("Number of params: {}", helper.params().len());
+    check_param_count(helper, template_name, 2)?;
+    trace!("params: {:?}", helper.params());
+
+    // Validate the values given to the template
+    trace!("Getting the desired OCI spec section to render in THIS call from the first param");
+    let current_oci_spec_section = get_param(helper, 0)?;
+    let current_oci_spec_section_was_null: bool = current_oci_spec_section.is_null();
```

Comment:
If we keep the "section" parameter then you should bail from the function here with an error.